ktransw.py

```diff
@@ -243,7 +243,7 @@ def main():
                         hdr_path = os.path.join(hdr_dir, hdr_path)
                         logger.debug("Found {0} in '{1}'".format(hdr, hdr_dir))
 
-                    except ValueError as e:
+                    except (ValueError) as e:
```

Comment:
A quick Google doesn't explain to me why the parentheses should be added here. Is this a Python3 best practice?

Reply:
I believe it was throwing an error without it. I had to google that myself. Should double check though.